com.unity.template-universal/Packages/com.unity.template.universal/package.json

```diff
@@ -7,6 +7,50 @@
   "host": "hub",
```

Comment:
It's an update for 21.2 but the minimum version is set to 22.1.
Was it updated using the 22.1 version?

Reply:
this template update is for master. The backport will be for 21.2